projects/js-packages/script-data/src/utils.ts

```diff
@@ -9,6 +9,20 @@ export function getScriptData() {
 	return window.JetpackScriptData;
 }
 
+/**
+ * Get the blog ID from the script data.
+ *
+ * @return {number|null} The blog ID or null if not available.
+ */
+export function getJetpackBlogId(): number | null {
```

Comment:
It seems like I am not able to use this function in current PR but still added it so that it can be used in future.